pages/css/frameworks.md

```diff
@@ -9,17 +9,30 @@ sidenav: css
 
 Sometimes, projects utilize other CSS frameworks such as:
 
-1. [Bourbon](https://www.bourbon.io/)
-2. [BassCSS](https://www.basscss.com/)
+- [Bourbon](https://www.bourbon.io/) (particularly well-suited to working with Sass)
+- [BassCSS](https://www.basscss.com/) (atomic CSS framework)
 
 These frameworks were chosen because they're relatively unopinionated about
 design decisions while still providing the helpers that make frameworks
-essential to fast and accurate frontend work, for example, solutions for
-responsive design, grids, and common design patterns. In addition, both
+essential to fast and accurate front-end work (for example, solutions for
+responsive design, grids, and common design patterns). In addition, both
 frameworks, through modular design and excellent documentation, make it easy
 for the designer or developer to only use the parts that they need, rather than
 including a hefty library.
 
+## Considerations for choosing a CSS framework
+
+When choosing a framework for your project, the following questions may be helpful in assessing your options:
+
+- What does it offer that the U.S. Web Design System doesn't?
+- How does it support responsive design?
+- Are patterns for more complex interactions (accordions, tabs, menus) accessible?
+- Does it allow for straightforward customization?
+- Is it mature and currently supported?
+- Does it support older browsers and provide fallbacks for newer techniques when necessary?
+- Is the library size reasonable?
+- Do other developers and designers on your team know how to work with this framework?
```

Comment:
Do these seem reasonable? What else should go in this list?

Reply:
I'm of the opinion that we should revamp this list to strongly suggest starting with the USWDS unless you have really compelling reasons not to, and to strongly suggest NOT introducing Yet Another Framework. Even in the cases where BassCSS has been used already, I know there's been a fair bit of follow-up discussion wondering why it was essentially reproducing USWDS. If we think is the right basis for the front-end on government sites (and I think that's safe to say) then I think we should be a bit firmer in that opinion.